code-challenges.test.js

```diff
@@ -50,3 +91,28 @@ const hand4 = [7, 2, 7, 2, 7]
 // Expected output: true
 
 // b) Create the function that makes the test pass.
+// const full house will have the array as the param
+// will sort and if certain index locations equal certain values that equal a fullhouse it will determine its full house.
+
+})
+})
+const fullHouse = (houseArr) => {
+    arr = arr.sort()
+    if (arr[0] === arr[1]) {
+    if (arr[0] === arr[2]) {
+         if (arr[3] == arr[4]) {
+                return true
+            }
+        } else if (arr[2] === arr[3]) {
+            if (arr[2] ===arr[4]) {
+                return true
+            } else {
+                return false 
+            }
+        } else {
+            return false 
+        }
+    } else {
+        return false
+    }
+}
```

Comment:
great job with the logic again!  The only issue with your code is that you pass houseArr as parameter, but then use arr to sort.  Also, incomplete test.  

Overall, for js challenges your logic looks good.  You could improve your pseudocode by giving more details of how you will break things down.